ui/src/Signin.tsx

```diff
@@ -68,12 +68,20 @@ export class Signin extends PureComponent<Props, State> {
   private checkForLogin = async () => {
     try {
       await client.users.me()
+      this.setState({loading: RemoteDataState.Done})
```

Comment:
do we need a loading state before the await?

Reply:
@ischolten ahh it looks like it was transitioning from `NotStarted` to `Done` before I can add a loading state.